.eslintrc.yml

```diff
@@ -2,6 +2,15 @@ root: true
 
 plugins:
     - node
+    - prettier
 extends:
     - "eslint"
     - "plugin:node/recommended"
+    - "prettier"
+rules:
+    prettier/prettier:
+    - error
+    - tabWidth: 4
+      printWidth: 120
```

Comment:
Hmm please could explain this choice? The default of 80 in prettier was chosen for a reason, and increasing it by 50% seems unnecessary